build-tools/script/diff2html.js

```diff
@@ -9,33 +9,56 @@ const EXT = '*.hbs';
 const questions = [
   {
     type: 'list',
-    name: 'type',
+    name: 'from',
     message:
-      'The script will generate diff reports for handlebars templates.\n What do you want to use as a diff to the current commit?',
+      'The script will generate diff reports for handlebars templates.\n What do you want to use as a first part of comparison?',
     choices: [
-      {
-        name: 'Commit hash',
-        value: 'hash',
-      },
       {
         name: 'Master branch',
         value: 'master',
       },
+      {
+        name: 'Commit hash or branch name',
```

Comment:
or tag :)

Reply:
We can simplify it and keep only <commit> like in official documentation.